tools/system_message.toit

```diff
@@ -46,13 +47,15 @@ main args:
   prog_name := "system_message"
   if not 1 <= args.size <= 3: usage prog_name
   snapshot := args[0]
-  if not file.is_file snapshot:
-    pipe.print_to_stderr "No such snapshot file: $snapshot"
-    usage prog_name
-  snapshot_content := file.read_content snapshot
-  if not SnapshotBundle.is_bundle_content snapshot_content:
-    pipe.print_to_stderr "Not a snapshot file: $snapshot"
-    usage prog_name
+  snapshot_content := null
+  if not snapshot.ends_with "00000000-0000-0000-0000-000000000000.snapshot":
```

Comment:
This feels a bit hacky. You're letting the Go code get away with passing a non-existing file to tools/system_message as long as it has a name you can recognize. It would be more natural to let the snapshot argument be optional and do the filtering in the Go code. Is it obvious which messages you can decode without a snapshot?